apps/frontend/src/app/poll/services/choose-date.service.ts

```diff
@@ -101,15 +101,16 @@ export class ChooseDateService {
       },
       meta: {
         tmpEvent: true,
+        note: note !== undefined ? note : undefined,
```

Comment:
`note !== undefined ? note : undefined` does nothing, so you can just use:
```suggestion
        note,
```